taxi/tests/test_admin.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+
+
+class AdminTestCase(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.admin_user = get_user_model().objects.create_superuser(
+            username="admin",
+            password="testadmin"
+        )
+        self.client.force_login(self.admin_user)
+        self.driver = get_user_model().objects.create_user(
+            username="driver",
+            password="testadmin",
+            license_number="DHF12342",
+        )
+
+    def test_driver_license_number_listed(self):
+        """"
+        Test that driver's license is listed on admin page
+        :return:
+        """
+        url = reverse("admin:taxi_driver_changelist")
+        res = self.client.get(url)
+        self.assertContains(res, self.driver.license_number)
+
+    def test_driver_detail_license_number_listed(self):
+        """"
+        Test that driver's license is on driver detail admin page
+        :return:
+        """
+        url = reverse("admin:taxi_driver_change", args=[self.driver.id])
+        res = self.client.get(url)
+        self.assertContains(res, self.driver.license_number)
+
+    def test_driver_detail_first_last_name_listed(self):
+        """"
+        Test that first, last name and driver's license is on driver detail admin page
+        :return:
+        """
```

Comment:
Similar to the previous comments, there is an extra double quote at the beginning of the docstring. It should be corrected to a triple quote to properly format the docstring.